src/main.ts

```diff
@@ -692,13 +692,13 @@ class GeneralSettingsTab extends PluginSettingTab {
                 );
         } else {
             taskDtFormat.setDesc(
-                'Only available when "Automatic Task Completion Tracking" is enabled and "Use Emoji Shorthand for Completion" is disabled.'
+                'Only available when "automatic task completion tracking" is enabled and "use emoji shorthand for completion" is disabled.'
```

Comment:
And here as well. Hmm, I guess we might wanna refactor settings at some point and make them conditional? Meh, I guess blacksmithgu is going to deprecate this at some point anyway.